### PR TITLE
Enable prettier for Admin SCSS

### PR DESCRIPTION
**Pull request by dacook**
#### What? Why?

In admin_style_v2, we enabled prettier to ensure the new code was up to standard. 

If this is our goal, we can start prettifying all our CSS code. 

This PR enables prettier for the entire `app/webpacker/css/admin/` directory. Unfortunately Prettier won't let you fix one rule at a time (https://stackoverflow.com/q/63813336/421243), so I'll break it into file chunks. 
There were a lot of changes, so I didn't do all files yet.

The prettier in the pre-commit hook and CI only checks _changed_ files, which is why it still passes even though I've not fixed all files. 

It's very easy to automatically fix files, so I'll follow that up with a second PR. IE:
```
yarn prettier --write app/webpacker/css/admin/
```

#### What should we test?
All changes are considered "safe", that is there should be no change to the styles. However it would be prudent to briefly check that a few screens on the admin interface look right.

#### Release notes

Changelog Category:  Technical changes

The title of the pull request will be included in the release notes.

#### Post-merge 
Update the rest of the admin files.

#### Documentation updates

We can create a new basic wiki page about prettier (which covers JS and CSS)
Also I think we can update the [Code-Conventions](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Code-Conventions) page to make it more of a landing page, pointing to rubocop and prettier as sub-pages.
Also Code-Conventions can be updated, 
  * the public_send rule has been enabled since 2018 and doesn't need to be mentioned (https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.rubocop_styleguide.yml#L71-L72)
   * I'm pretty sure the `have_not` recommendation is no longer relevant.

### CODE REVIEW

File: app/webpacker/css/admin/components/messages.scss [hunk 1]
```diff
@@ -34,19 +34,29 @@
...
+
+    &.notice {
+      background-color: rgba($color-notice, 0.8);
+    }
+    &.success {
+      background-color: rgba($color-success, 0.8);
+    }
+    &.error {
+      background-color: rgba($color-error, 0.8);
+    }
```
Comment (dacook) [inline app/webpacker/css/admin/components/messages.scss:51]: Arguably these lines are now less readable, but Prettier doesn't provide any options to choose a different format.

Comment (jibees) [reply app/webpacker/css/admin/components/messages.scss:51]: > but Prettier doesn't provide any options to choose a different format.

For me, this is the force of Prettier: to be (almost) full opinionated.

**Review by jibees — APPROVED** [review]
Seems perfect!

**Review by rioug — APPROVED** [review]
Looks good :+1:  
That double quote rule remind me we should also apply the same rule in Rubocop, I think it's currently disable.

**Comment by drummer83** [comment]
Hi @dacook,
Thanks for this one!

I have looked at quite a few admin pages and also quickly checked the shopfront as well. Here are some screenshots for reference after staging.

image
image
image
image
image
image
image
image
image
image


I couldn't see any design issues which could be related to these changes. I think we can go ahead and merge this. :muscle: 

Thanks again! :pray:

**Pull request merged**